AliveSFU/AliveSFU/MyProgressController.swift

```diff
@@ -21,12 +21,12 @@ class MyProgressController: UIViewController {
     }
     @IBAction func showPopup(_ sender: Any) {
```

Comment:
The following is a better implementation:

<code>
@IBAction func showPopup(_ sender: Any) {
   
        let popOverVC = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "tilePopUpID") as! ExerciseTileViewController
        popOverVC.modalPresentationStyle = .popover

        let popover = popOverVC.popoverPresentationController
        popover?.permittedArrowDirections = .init(rawValue: 0)
        popover?.sourceView = self.view
        popover?.sourceRect = CGRect(x: self.view.bounds.midX, y: UIScreen.main.bounds.midY, width: 1, height: 1)
        popover?.delegate = self

        self.present(popOverVC, animated: true)
    }

    func adaptivePresentationStyle(for controller: UIPresentationController) -> UIModalPresentationStyle {
        return .none
    }
</code>